### DISCUSSION TITLE
Welcome to TextEditor Discussions!

### DISCUSSION THREAD

**Discussion by TerabyteTB** [TerabyteTB/WhirlpoolEditor#2] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by akitts** [comment]
Hi there, I am new to open source and looking to get a start. I am not entirely sure where to start at, but if there is anywhere you would like me to focus effort on I am happy to help! Feel free to reach out and let me know!

**Reply by TerabyteTB** [reply]
This is actually a discussion template generated by GitHub, but of course, you can help if you want! :)

**Reply by TerabyteTB** [reply]
@akitts There’s one thing you can help. You might want to help test it on Windows. (I hard-coded all the paths so Linux doesn’t work yet)
